crypto/rsa_extra/rsa_asn1.c

```diff
@@ -153,6 +153,32 @@ int RSA_public_key_to_bytes(uint8_t **out_bytes, size_t *out_len,
 // RSAPrivateKey structure (RFC 3447).
 static const uint64_t kVersionTwoPrime = 0;
 
+// Distinguisher for stripped ACCP RSA private keys, sets zeroed values to NULL
+// because ASN.1 treats absent values as 0, but post-parsing validation logic
+// expects absent values to be NULL. Returns 1 if ACCP stripped private key, 0
+// otherwise.
+static int detect_stripped_accp_private_key(RSA *key) {
```

Comment:
```suggestion
static int detect_stripped_jca_private_key(RSA *key) {
```

Reply:
i have no issue with that. perhaps we can consolidate discussion with @SalusaSecondus around the naming issue on [this thread](https://github.com/awslabs/aws-lc/pull/436#discussion_r850827317)?